### PR TITLE
Add required prop to fields in PreviousExposure form

### PR DESCRIPTION
**Pull request by juliatan**
# Description

Add required props to the PreviousExposure to COVID form.

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

Simulator Screen Shot - iPhone 11 - 2021-06-29 at 16 22 41

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if needed)

## Checklist (for reviewers)

- [ ] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Out of scope and potential follow-up

Need to refactor to a functional component so that we can reset the form if a user clicks back on navbar after submitting. 
Ticket: https://www.notion.so/joinzoe/Product-Eng-Board-ca9e8b1b8926481f8c94cb01a88d567d?p=5c5011dd2c9f4b86a01f1db55820f8bd

### CODE REVIEW

File: src/features/patient/PreviousExposureScreen.tsx [hunk 1]
```diff
@@ -33,7 +34,7 @@ interface IYourHealthData {
 
 const initialFormValues = {
   classicSymptoms: 'no',
-  pastSymptomsChanged: 'no',
+  pastSymptomsChanged: 'much_better',
```
Comment (aikewoody) [inline src/features/patient/PreviousExposureScreen.tsx:37]: Is this change intentional?

File: src/features/patient/PreviousExposureScreen.tsx [hunk 2]
```diff
@@ -290,8 +296,10 @@ export default class PreviousExposureScreen extends React.Component<HealthProps,
                   ) : null}
                 </View>
 
-                <BrandedButton onPress={props.handleSubmit}>{i18n.t('next-question')}</BrandedButton>
-              </Form>
+                <BrandedButton enable={props.isValid} onPress={props.handleSubmit}>
```
Comment (aikewoody) [inline src/features/patient/PreviousExposureScreen.tsx:299]: Should we also check for `props.dirty` like on the other forms?

File: src/features/patient/PreviousExposureScreen.tsx (hunk 1 above)
Comment (juliatan) [reply src/features/patient/PreviousExposureScreen.tsx:37]: Yep, this is intentional - `no` doesn't exist as an option.

File: src/features/patient/PreviousExposureScreen.tsx (hunk 2 above)
Comment (juliatan) [reply src/features/patient/PreviousExposureScreen.tsx:299]: It's not needed in this case because there's just one text input field here which is optional. :)

Comment (aikewoody) [reply src/features/patient/PreviousExposureScreen.tsx:299]: How about this line?
```
                      <GenericTextField
                        required
                        formikProps={props}
                        keyboardType="numeric"
                        label={i18n.t('label-past-symptoms-days-ago')}
                        name="pastSymptomsDaysAgo"
                      />
```

Comment (juliatan) [reply src/features/patient/PreviousExposureScreen.tsx:299]: It's hidden unless a specific yes/no field is toggled which means that form will have to be dirty by default to be valid.
Upon form load, there's only one yes/no toggle which auto defaults to "no", so theoretically, the form should be enabled, even if user doesn't "dirty" it.

**Review by aikewoody — APPROVED** [review]
